config.example.yaml

```diff
@@ -76,6 +76,8 @@ logging:
   instance_name: block-header-service
   # Flag for enabling origin parameter in logs
   origin: false
+  # Release mode should be enabled in "production"
+  release_mode: false
```

Comment:
Can we call it `gin_release_mode`? Because it's easy to get confused 😄

Reply:
Yes we can 👍 Please re-review this to ensure it's no longer confusing.